modules/react/tooltip/lib/useTooltip.tsx

```diff
@@ -133,19 +133,33 @@ export function useTooltip<T extends Element = Element>({
 
   const visible = popupModel.state.visibility !== 'hidden';
 
+  const targetProps: {
+    'aria-describedby'?: string;
+    'aria-label'?: string;
+    onMouseEnter: typeof onOpenFromTarget;
+    onMouseLeave: typeof onHide;
+    onMouseDown: typeof onMouseDown;
+    onFocus: typeof onFocus;
+    onBlur: typeof onHide;
+  } = {
```

Comment:
I don't like the overhead of re-typing an object. I'll rewrite to prevent the need for it.